uniffi_macros/src/error.rs

```diff
@@ -79,7 +79,7 @@ fn error_ffi_converter_impl(ident: &Ident, enum_: &DataEnum, attr: &ErrorAttr) -
 
 // FfiConverters for "flat errors"
 //
-// These are errors where we only expose the variants, not the data.
+// These are errors where we only expose to_string of the variants, not the data.
```

Comment:
This one still seems a bit unclear to me.  How about "These are errors where we only lower the `to_string()` value, rather than any assocated data".